### PR TITLE
Fall-back to own llvm-symbolizer if one from archive fails.

### PR DESCRIPTION
**Pull request by inferno-chromium**
Fixes https://github.com/google/clusterfuzz/issues/2214

### CODE REVIEW

File: src/python/system/environment.py [hunk 1]
```diff
@@ -332,13 +333,25 @@ def get_llvm_symbolizer_path():
...
+  if llvm_symbolizer_path and os.path.exists(llvm_symbolizer_path):
+    # Make sure that llvm symbolizer binary is executable.
+    os.chmod(llvm_symbolizer_path, 0o750)
+
+    return_code = subprocess.call(
+        [llvm_symbolizer_path, '--help'],
+        stdout=subprocess.DEVNULL,
+        stderr=subprocess.DEVNULL)
+    if return_code == 0:
+      # llvm-symbolize works, return it.
```
Comment (oliverchang) [inline src/python/system/environment.py:345]: nit: log_error so we know about it?

**Review by oliverchang — APPROVED** [review]
thanks!

File: src/python/system/environment.py (hunk 1 above)
Comment (inferno-chromium) [reply src/python/system/environment.py:345]: environment is such a self-contained module, that is why skipped adding logging dependency to avoid cycles.

Comment (oliverchang) [reply src/python/system/environment.py:345]: Ah right, thanks for clarifying.

**Pull request merged**